seong-geun/app.js

```diff
@@ -115,22 +129,55 @@ app.get("/u_posts", async (req, res) => {
    `,
     [userId]
   );
+
   res.status(200).json({ data: userspost });
-  const { id, profile_image, username, age, email } = req.body;
+});
 
+// 게시글 수정하기
+app.patch("/m_posts", async (req, res) => {
+  const { postingUserId, postingContent } = req.body;
   await appDataSource.query(
-    `INSERT INTO users(
-      id,
-      profile_image,
-      username,
-      age,
-      email
-		) VALUES (?, ?, ?, ?, ?);
-	`,
-    [id, profile_image, username, age, email]
+    `UPDATE
+        posts
+    SET
+        posting_content = ?
+    WHERE
+        posts.user_id = ? 
+    `,
+    [postingContent, postingUserId]
   );
 
-  res.status(201).json({ message: "userCreated" });
+  const m_userspost = await appDataSource.query(
+    `SELECT
+        users.id as userId,
+        users.username as userName,
+        posts.id as postingId,
+        posts.title as postingTitle,
+        posts.posting_content as postingContent
+    
+    FROM 
+        users
+    LEFT JOIN posts
+    ON users.id = posts.user_id
+    `
+  );
+
+  res.status(200).json({ data: m_userspost });
```

Comment:
```suggestion
  return res.status(201).json({ data: m_userspost });
```
결국엔 해당 API도 내부 로직이나, http method를 보면 단순 응답을 의미하는 200보다는 더 적합한 status code를 활용하는게 좋을 것 같습니다.